editoast/src/core/mod.rs

```diff
@@ -264,7 +265,7 @@ impl Error {
     }
 }
 
-#[derive(Debug, Deserialize, PartialEq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
```

Comment:
As discussed earlier, we might not want to expose (e.g. derive `ToSchema`) for errors coming from `core`.